02_building_containers/install_flash_attn.py

```diff
@@ -1,8 +1,23 @@
+# ---
+# cmd: ["modal", "run", "02_building_containers/install_flash_attn.py::main"]
+# ---
+
+# # Install Flash Attention on Modal
+# FlashAttention is an optimized CUDA library for Transformer
+# scaled-dot-product attention. Dao AI Lab now publishes pre-compiled
+# wheels, which makes installation quick.  This script shows how to
+#
+# 1. Pin an exact wheel that matches CUDA 12 / PyTorch 2.6 / Python 3.13.
+# 2. Build a Modal image that installs torch, numpy, and FlashAttention.
+# 3. Launch a GPU function to confirm the kernel runs on a GPU.
+#
 import modal
 
 app = modal.App("example-install-flash-attn")
 
-flash_attn_release = (  # find releases at https://github.com/Dao-AILab/flash-attention/releases
+# Here we specify an exact release wheel. You can find
```

Comment:
Because the comment is on a new line, it will be rendered as text.

I would stick with the existing comment.